l10n_it_fatturapa_in/wizard/link_to_existing_invoice.py

```diff
@@ -17,3 +17,16 @@ def link(self):
         if len(active_ids) != 1:
             raise UserError(_("You can select only one XML file to link."))
         self.invoice_id.fatturapa_attachment_in_id = active_ids[0]
+        # extract pdf if attached
+        fatturapa_attachment_obj = self.env['fatturapa.attachment.in']
+        for fatturapa_attachment_id in active_ids:
+            fatturapa_attachment = fatturapa_attachment_obj.browse(
+                fatturapa_attachment_id)
+            fatt = self.get_invoice_obj(fatturapa_attachment)
+            for FatturaBody in fatt.FatturaElettronicaBody:
+                self.invoice_id.set_einvoice_amount(FatturaBody)
```

Comment:
Qui i vari campi `e_invoice_amount` vengono sovrascritti per ogni fattura del file XML.
Questo verrà corretto in v10 da https://github.com/OCA/l10n-italy/pull/1209 insieme ad altre nuove funzionalità.

Non saprei come sia meglio procedere, forse si potrebbe mergiare questa che è parzialmente bacata e aspettare che poi venga mergiata e portata in v11 #1209